torch/testing/_internal/common_utils.py

```diff
@@ -2499,3 +2499,10 @@ def has_breakpad() -> bool:
         return True
     except RuntimeError as e:
         return False
+
+def find_library_location(condition: bool, lib_name: str) -> Path:
+    if condition:
+        torch_root = Path(torch.__file__).resolve().parent
+        return torch_root / 'lib' / lib_name
+    torch_root = Path(__file__).resolve().parent.parent.parent
```

Comment:
What if we get rid of condition and just check if file exists at the path?
```suggestion
    torch_root = Path(torch.__file__).resolve().parent
    path = torch_root / 'lib' / lib_name
    if os.path.exists(path):
        return path
    torch_root = Path(__file__).resolve().parent.parent.parent
```